health_check.go

```diff
@@ -1,65 +1,80 @@
 package main
 
 import (
-	"encoding/json"
-	"fmt"
-	"net/http"
+	"github.com/alphagov/govuk_crawler_worker/healthcheck"
 
```

Comment:
TBH I barely ever look at import statements because [`goimports`](https://godoc.org/golang.org/x/tools/cmd/goimports) does it all for me.  It just failed to remove the blank line this time.